desk/app/bait.hoon

```diff
@@ -88,53 +105,97 @@
     :_  this
     =/  full-line=request-line:server  (parse-request-line:server url.request)
     =/  line
-      ?:  ?=([%lure @ @ *] site.full-line)
+      ?:  ?=([%lure @ *] site.full-line)
         t.site.full-line
       ?:  ?=([@ @ *] site.full-line)
         site.full-line
       !!
     ?+    method.request  (give not-found:gen:server)
-        %'GET'
-      ?:  ?=([%bait %who ~] line)
-        (give (json-response:gen:server s+(scot %p our.bowl)))
-      =/  inviter  (slav %p i.line)
-      =/  token  i.t.line
-      =/  =metadata:reel  (fall (~(get by token-metadata) [inviter token]) *metadata:reel)
-      ?:  ?=([@ @ %metadata ~] line)
-        (give (json-response:gen:server (enjs-metadata metadata)))
-      (give (manx-response:gen:server (landing-page metadata)))
+      %'GET'  (get-request line)
+    ::
         %'POST'
-      =/  inviter  (slav %p i.line)
-      =/  token  i.t.line
       ?~  body.request
+        ~&  "body not found"
         (give not-found:gen:server)
       ?.  =('ship=%7E' (end [3 8] q.u.body.request))
+        ~&  "ship not found in body"
         (give not-found:gen:server)
       =/  joiner  (slav %p (cat 3 '~' (rsh [3 8] q.u.body.request)))
-      :*  :*  %pass  /bite  %agent  [inviter %reel]
-              %poke  %reel-bite  !>([%bite-1 token joiner inviter])
-          ==
-          :*  %pass  /bite  %agent  [our.bowl %reel]
-              %poke  %reel-bite  !>([%bite-1 token joiner inviter])
-          ==
-          (give (manx-response:gen:server (sent-page joiner)))
-      ==
+      =;  [=bite:reel inviter=(unit ship)]
+        ?~  inviter
+          ~&  "inviter not found"
+          (give not-found:gen:server)
```

Comment:
Should these printfs maybe go into the 404 response bodies instead? Or wherever would make them show up in the hosting-side logs.